Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -1285,6 +1314,36 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
     return .visitChildren
   }
 
+  public override func visit(_ node: UnavailableFromAsyncArgumentsSyntax) -> SyntaxVisitorContinueKind {
+    if shouldSkip(node) {
+      return .skipChildren
+    }
+
+    if let token = node.unexpectedBetweenMessageLabelAndColon?.onlyToken(where: { $0.tokenKind.isIdentifier }),
+      node.messageLabel.presence == .missing
+    {
+      addDiagnostic(
+        node,
+        MissingNodesError(missingNodes: [Syntax(node.messageLabel)]),
```

Comment:
Why do we need this special diagnostic handling? Wouldn’t we get the same from the default handling?

Reply:
It did replace "" in front of the the identifier instead of wrapping the identifier in "".
So `message: bar` will become `message: ""bar` instead of `message: "bar"`

Reply:
That seems like we’re parsing it incorrectly. I would expect `bar` to be in the `segments` child of `StringLiteralExprSyntax` with the second quote after it. But it seems like `bar` got stuck somewhere after `closeQuote` if it produces `message: ""bar` as the fixed source.

Reply:
Ah oh! 
I was wrong here. 
This fixes that `@_unavailableFromAsync(otherLabel: "abc")` bot becomes `@_unavailableFromAsync(otherLabel message: "abc")`